.gitignore

```diff
@@ -23,6 +23,7 @@ cabal.project.local
 .HTF/
 # Stack
 .stack-work/
+*.yaml.lock
```

Comment:
seems a bit too of a wildcard. Which file do you want to exclude? `stack.yaml.lock` or does it generate `stack-version.yaml.lock` files as well? (I'm not sure).

Reply:
Previously there was `stack.yaml.lock` as well as `stack-*.yaml.lock`. I resorted to `*.yaml.lock` in order to tackle both at once but I've completely removed `stack.yaml` after the latest commit. So I've updated the wildcard to only track `stack-*` lock files.